yt/frontends/sph/io.py

```diff
@@ -14,6 +14,8 @@ class IOHandlerSPH(BaseParticleIOHandler):
     This exists to handle particles with smoothing lengths, which require us
     to read in smoothing lengths along with the the particle coordinates to
     determine particle extents.
+
+    At present this is non-functional.
```

Comment:
What'd you mean by "non-functional" here? Lots of frontends inherit from this class.

Reply:
I *think* what I meant was that it was a base class that wasn't used?  Or maybe, it was meaning that the smoothing lengths were not globally implemented?

Reply:
ah, gotchya. Cause this base class is not actually adding a global smoothing length implementation -- all the classes inheriting from this implement their own. It's still a useful organizational feature as is though! And I've done some work that actually adds stuff to this base class (don't remember exactly what work -- either one of my open PRs or stuff I haven't pushed). But in any case, I'm fine with this comment :)